client/src/lib/dependency-injection/graphql/buildBaseQuery.js

```diff
@@ -12,7 +12,7 @@ const getQueryType = ({ queryType }) => queryType;
 
 const buildReadQuery = (tag = defaultTag) => (
   tag`${getQueryType} ${getOperationName}${getVariables} {
-    ${getQueryName}(${getParams}) {
+    ${getQueryName}${getParams} {
```

Comment:
This would have ended up as `readMyModel((param: $param)) {` which would result in a syntax error.